### PR TITLE
Upgrade to Masonite 3

### PR DESCRIPTION
**Pull request by vaibhavmule**
cc @josephmancuso

### CODE REVIEW

File: python/masonite/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-masonite>=2.3.1,<2.4.0
+masonite>=3.0,<4.0
```
Comment (waghanza) [inline python/masonite/requirements.txt:1]: that's a huge range. This one not guarantee that we are in 3.0, right ?
```suggestion
masonite>=3.0.0,<3.1.0
```

**Review by waghanza — APPROVED** [review]

File: python/masonite/requirements.txt (hunk 1 above)
Comment (vaibhavmule) [reply python/masonite/requirements.txt:1]: @waghanza I don't have an idea about the roadmap, @josephmancuso please guide here.

Comment (waghanza) [reply python/masonite/requirements.txt:1]: @vaibhavmule what I meant is that we show version on `README` (based on `config.yaml`). in order to be in sync, we must restrict the range

Comment (josephmancuso) [reply python/masonite/requirements.txt:1]: I'm 99% sure Masonite 4 will be released after Masonite 3 but putting `masonite>=3.0.0,<3.1.0` is a safe bet for now.

Comment (waghanza) [reply python/masonite/requirements.txt:1]: that leave 1% uncertainty @josephmancuso 😛

**Pull request merged**